.github/workflows/ci.yml

```diff
@@ -25,9 +25,21 @@ jobs:
           mix local.rebar --force
           mix local.hex --force
           mix deps.get
-      - name: Compile Code
-        run: mix compile --warnings-as-errors
       - name: Ensure Code is Formatted
         run: mix format --check-formatted
```

Comment:
`check formatted` is ran before the compilation because it doesn't require compiling, so CI will fail faster in these cases